### ISSUE TITLE
false positive on install.sh

### ISSUE THREAD

**Issue by Sispheor** [rust-lang/rustlings#118]
In my case, cargo is not installed, but I got `SUCCESS: Cargo is installed`

```
curl -L https://git.io/rustlings | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  1350  100  1350    0     0   1839      0 --:--:-- --:--:-- --:--:--  1839
Let's get you set up with Rustlings!
Checking requirements...
SUCCESS: Git is installed
bash: line 15: rustc: command not found
SUCCESS: Rust is installed
bash: line 24: cargo: command not found
SUCCESS: Cargo is installed
Cloning Rustlings at rustlings/...
Traceback (most recent call last):
  File "<string>", line 1, in <module>
KeyError: 'tag_name'
Checking out version ...
error: pathspec 'tags/' did not match any file(s) known to git.
Installing the 'rustlings' executable...
bash: line 43: cargo: command not found
bash: line 45: rustlings: command not found
All done! Run 'rustlings' to get started.
```

**Comment by whoizit** [comment]
i've installed fine, but something wrong
```
$ rustc -V
rustc 1.32.0
```


```
$ rustlings

       welcome to...                      
                 _   _ _                  
  _ __ _   _ ___| |_| (_)_ __   __ _ ___  
 | '__| | | / __| __| | | '_ \ / _` / __| 
 | |  | |_| \__ \ |_| | | | | | (_| \__ \ 
 |_|   \__,_|___/\__|_|_|_| |_|\__, |___/ 
                               |___/      

thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/libcore/result.rs:1009:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.

```

**Comment by whoizit** [comment]
ok, I should cd to rustlings directory and after that i should rust rustlings. strange

**Comment by komaeda** [comment]
@whoizit yeah, we should add some kind of warning that you should run rustlings from inside of the directory

**Comment by komaeda** [comment]
closed in version 1.1.0! https://github.com/rust-lang/rustlings/releases/tag/1.1.0
